cypress/objects/timeLoginPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoginPage {
+    visit() {
+        cy.visit('/')
+    }
+
+    getURL() {
+        return cy.url().should('include', '/time-logging')
+    }
+
+    getPageTitle() {
```

Comment:
According to the name of the function, the function should only get the element and not assert that it contains 'Timesheets', so I'd drop the 'contains' part and add it in the test cases instead of this function's definition :)

Reply:
This comment was not addressed